src/features/student/locales/en.json

```
@@ -364,8 +357,19 @@
             "success": "Your trace has been successfully added."
           }
         },
-        "studentToolsTracesActionButtons": {
-          "addTrace": "ADD A TRACE TO MY LIBRARY"
+        "studentToolsTracesViewContainer": {
+          "pagination": {
+            "bottom": {
+              "ariaLabel": "Bottom pagination"
+            },
+            "top": {
+              "ariaLabel": "Top pagination"
+            }
+          }
+        },
+        "studentToolsTracesViewTabs": {
+          "associated": "My associated traces",
+          "unassociated": "My unassociated traces"
```

Comme mis dans un autre commentaire, fais la gestion du pluriel ici, ça peut être bien
```suggestion
          "associated": "My associated traces ({count})",
          "unassociated": "My unassociated traces ({count})"
```

Idem pour fr.json aussi ;)